charts/das/templates/_helpers.tpl

```diff
@@ -62,13 +62,22 @@ Create the name of the service account to use
 {{- end }}
 
 {{- define "startupProbe" -}}
-curl "http://localhost:{{ .Values.configmap.data.http.port }}{{ .Values.configmap.data.http.rpcprefix }}" -H "Content-Type: application/json" \
-         -sd "{\"jsonrpc\":\"2.0\",\"id\":0,\"method\":\"eth_syncing\",\"params\":[]}" \
-         | jq -ne "input.result == false"
+curl "http://localhost:{{ index .Values.configmap.data "rpc-port" }}" -X POST \
```

Comment:
Looks like`.Values.configmap.data "rpc-port"` is missing from the values YAML

Reply:
https://github.com/OffchainLabs/community-helm-charts/blob/5005b0d952b773936ea0085d6e7290fe94326f7d/charts/das/values.yaml#L153

Reply:
That is intentional. That values file is a test case for a das mirror which would not have the rpc enabled.